HISTORY.rst

```diff
@@ -3,6 +3,11 @@
 History
 -------
 
+0.4.22 (2019-02-05)
+___________________
+
+* Changed to able to get/update 'aliases' parameter of HOST record for DNS
```

Comment:
can this changes be removed, because we are planning to merge another pr and bump the version.

Reply:
Thank you for your review.

> we are planning to merge another pr and bump the version.

I got it. After merging it, I'll merge latest master branch and resolve conflicts of these files (`HISTORY.rst` and `infoblox_client/__init__.py`) properly.
Please go ahead.

Reply:
Thank you for your review.

> we are planning to merge another pr and bump the version.
I got it. After merging it, I'll merge latest master branch and resolve conflicts of these files (`HISTORY.rst` and `infoblox_client/__init__.py`) properly.
Please go ahead.

Reply:
no i meant remove the History.rst from this pr, i will update it when we are planning to bump the version